### ISSUE TITLE
NTR: Glyoxylic acid high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8264]
New term request Glyoxylic acid high in blood (elthat_100703163715): Glyoxylate

**Comment by pnrobinson** [comment]
@MickeySegal
what is the clinical context?

**Comment by MickeySegal** [comment]
It is high in both ethylene glycol poisoning and in vitamin B1 deficiency.
https://karger.com/nef/article-abstract/48/3/248/215095/Glyoxylic-Acid-in-Ethylene-Glycol-Poisoning?redirectedFrom=PDF
https://academic.oup.com/nutritionreviews/article-abstract/22/1/23/1934203?redirectedFrom=PDF

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000974
- label: Elevated circulating glyoxylic acid concentration
- parents: [Abnormal circulating monocarboxylic acid concentration (HP:0010996)](https://hpo.jax.org/app/browse/term/HP:0010996)
- pmid: 
- def: The concentration of glyoxylic acid in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8264
- synonyms:
